app/templates/components/wds-community-bar/local-navigation-level-2.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="wds-community-bar__navigation-back" {{action onBackButtonClicked}}>
+	{{svg 'wds-icons-menu-control-tiny' class='wds-icon wds-icon-tiny'}} {{i18n 'back'}}
+</div>
+{{#if firstLevelItem.href}}
+	<a
+		class="wds-community-bar__navigation-header"
+	   	href="{{firstLevelItem.href}}"
+		data-tracking-label="{{unless (equal firstLevelItem.href '#') firstLevelItem.tracking_label}}"
+		onclick={{action linkClicked}}
+	>
+		{{wds-label label=firstLevelItem.title}}
+	</a>
+{{/if}}
+{{#wds-list linked=true}}
```

Comment:
I wonder if we could redo this component so this would be possible (not now, just thinking :))

```
<WdsList @linked=true @collection=firstLevelItem.items as |secondLevelItem index|>
  <a href="{{secondLevelItem.href}}" data-tracking-label="{{unless (equal secondLevelItem.href '#') secondLevelItem.tracking_label}}" onclick={{action onSecondLevelClicked index secondLevelItem}}>
				{{wds-label label=secondLevelItem.title}}
				{{#if secondLevelItem.items}}
					{{svg 'wds-icons-menu-control-tiny' class='wds-icon wds-icon-tiny'}}
				{{/if}}
			</a>
</WdsList>